packages/swirl-components/src/components/flip-app-layout/flip-app-layout.css

```diff
@@ -0,0 +1,640 @@
+@import "../../styles/media-queries.css";
```

Comment:
this is quite a huge file, wouldn’t it make sense to split it up in viewports and import them in a „main“-file?

Reply:
Yes. But maybe split by base styles and transition styles, as the transitions cause most of the noise, I think.

Reply:
yup, makes sense. do you want to do it in this PR or in the update PR?

Reply:
Just committed the changes.